wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/LinkedListElement.java

```diff
@@ -15,9 +15,6 @@
   private LinkedListElement m_previous;
   private Command m_data;
 
-  public LinkedListElement() {
```

Comment:
https://stackoverflow.com/questions/243218/should-we-declare-a-public-constructor-when-the-class-is-declared-as-package-pri